p2p/enode/node.go

```diff
@@ -82,6 +82,20 @@ func (n *Node) UDP() int {
 	return int(port)
 }
 
+// RAFTPORT returns the RAFT PORT of the node
+func (n *Node) RAFTPORT() int {
```

Comment:
Why is the method RAFTPORT() in caps

Reply:
The method name for other attributes like UPD, TCP, ID and IP in p2p/enode/node.go are all in CAPS. They are following this convention for KEY names basically. I have followed the same style to keep it consistent.

Reply:
I think the reason why `UDP`, `TCP`, `IP` are in CAPS because they are all acronyms and `RAFTPORT` is not, hence it need not be in CAPS

Reply:
alright, changed it to RaftPort